src/DwitTech.AccountService.Data/Entities/UserStatus.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Data.Entities
+{
+    public enum UserStatus
```

Comment:
If there's no db set for this, then I think it should be moved to Models.

Reply:
> If there's no db set for this, then I think it should be moved to Models.

It is linked in my Models already, so the enum can be called from the entity